### PR TITLE
Feature/umbraco hackathon docs vscode

### PR DESCRIPTION
**Pull request by jacksorjacksor**
Here are my first few suggestions for the documentation change - there are a few more I want to make but I need to get better at using Umbraco first!!

### CODE REVIEW

File: 11/umbraco-cms/SUMMARY.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
     * [Install using .NET CLI](fundamentals/setup/install/install-umbraco-with-templates.md)
     * [Install using Visual Studio](fundamentals/setup/install/visual-studio.md)
     * [Local IIS With Umbraco](fundamentals/setup/install/iis.md)
-    * [Install Umbraco With Visual Studio Code](fundamentals/setup/install/install-umbraco-with-vs-code.md)
+    * [Install using Visual Studio Code](fundamentals/setup/install/install-umbraco-with-vs-code.md)
```
Comment (sofietoft) [inline 11/umbraco-cms/SUMMARY.md:17]: You need to also update the title of the article, when updating the link to it in the navigation 💪 
They are tied together, so if you don't do it, GitBook will change this back to use the article title again 😅

File: generic/contribute/getting-started.md [hunk 2]
```diff
@@ -33,6 +33,10 @@ The Umbraco Documentation is written using the MarkDown markup language. We have
 
 Learn how we structure and name files in the Umbraco documentation.
 
+## IDE / Text Editor
```
Comment (sofietoft) [inline generic/contribute/getting-started.md:36]: Perhaps instead we could call this "Writing docs locally" or something?

File: generic/contribute/getting-started.md [hunk 3]
```diff
@@ -33,6 +33,10 @@ The Umbraco Documentation is written using the MarkDown markup language. We have
...
+
+We recommend using Visual Studio Code for making changes to the documentation.
```
Comment (sofietoft) [inline generic/contribute/getting-started.md:38]: Added a littel more context to this sentence. Let me know what you think!

```suggestion
We recommend using a tool like Visual Studio Code for making changes to the documentation on your local machine.
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
Thanks a bunch for the PR @jacksorjacksor 💪 

I've made a couple of suggestions - let me know what you think!

**Review by sofietoft — COMMENTED** [review]
Thanks for considering the suggestions @jacksorjacksor !

I see there are a couple of warnings - one for a too-long sentence, and one for an acronym.
Would you be up for taking a shot at resolving those?
You can learn more about the guidelines here: https://docs.umbraco.com/welcome/contribute/style-guide

**Comment by jacksorjacksor** [comment]
100% - will look into after/during Umbraco Spark!

**Comment by sofietoft** [comment]
Thanks @jacksorjacksor !

I'll make sure this is merged in 💪

**Comment by jacksorjacksor** [comment]
YAY - thank you @sofietoft !!

**Pull request merged**